test/system/src/main/java/io/pravega/test/system/framework/services/docker/PravegaSegmentStoreDockerService.java

```diff
@@ -46,6 +47,7 @@
     private final URI zkUri;
     private final URI conUri;
     private final URI hdfsUri;
+    private final String image = Utils.isAwsExecution() ? "nautilus/pravega:" + PRAVEGA_VERSION : IMAGE_PATH + "/nautilus/pravega:" + PRAVEGA_VERSION;
```

Comment:
IMAGE_PATH is not required for AWS based execution; We could ensure IMAGE_PATH varies depending on the type of execution.

Reply:
done.